### ISSUE TITLE
`recitale build` throws an exception related to the themes directory

### ISSUE THREAD

**Issue by fr33tux** [recitale/recitale#1]
With recitale 2.0.0a0 on Ubuntu 21.10 with Python3.9, running `recitale build` throws the following exception:

> recitale build

```
Traceback (most recent call last):
  File "/usr/local/bin/recitale", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.9/dist-packages/recitale/recitale.py", line 807, in main
    templates = get_gallery_templates(theme, date_locale=date_locale)
  File "/usr/local/lib/python3.9/dist-packages/recitale/recitale.py", line 204, in get_gallery_templates
    Path(__file__).parent.joinpath("themes").listdir()
  File "/usr/local/lib/python3.9/dist-packages/path/__init__.py", line 474, in listdir
    return list(filter(match, (self / child for child in os.listdir(self))))
FileNotFoundError: [Errno 2] No such file or directory: Path('/usr/local/lib/python3.9/dist-packages/recitale/themes')
```

The behavior is the same if installed in a Virtualenv (`FileNotFoundError: [Errno 2] No such file or directory: Path('/home/xx/dev/blog_gallery/ve/lib/python3.9/site-packages/recitale/themes')`)

**Comment by QSchulz** [comment]
Hello @fr33tux,

Thank you for reporting this bug!

This should be fixed now in v2.0.0a1. Please upgrade recitale by running `python3 -m pip install -U recitale`.

Cheers,
Quentin

**Comment by fr33tux** [comment]
I confirm v2.0.0a1 fixes the issue, thanks! :clap:
